quinn/src/endpoint.rs

```diff
@@ -446,22 +471,25 @@ where
     S: proto::crypto::Session,
 {
     pub(crate) fn new(socket: UdpSocket, inner: proto::generic::Endpoint<S>, ipv6: bool) -> Self {
-        let recv_buf = vec![0; inner.config().get_max_udp_payload_size().min(64 * 1024) as usize];
+        let recv_buf =
```

Comment:
Do you maybe need to do anything for better alignment of buffers? I get `max_udp_payload_size` could have any arbitrary value

Reply:
Is there reason to believe buffer alignment matters for `recvmmsg`? We could round up if it's useful to.

Reply:
We probably would need to read kernel code to know for sure :-) 
It could make it more efficient for it to map/copy things, or it might not matter at all. I would maybe just round up to the next 8byte boundary per chunk.

Reply:
I'm inclined to avoid dubiously-justified magic for now in the spirit of maintenance.